mmv1/third_party/terraform/provider/provider_mmv1_resources.go.tmpl

```diff
@@ -33,11 +33,11 @@ var handwrittenDatasources = map[string]*schema.Resource{
 	"google_apphub_discovered_service":		    apphub.DataSourceApphubDiscoveredService(),
 	{{- if ne $.TargetVersionName "ga" }}
 	"google_backup_dr_management_server":				backupdr.DataSourceGoogleCloudBackupDRService(),
-	"google_backup_dr_backup":							backupdr.DataSourceGoogleCloudBackupDRBackup(),
 	"google_backup_dr_backup_plan_association":			backupdr.DataSourceGoogleCloudBackupDRBackupPlanAssociation(),
 	"google_backup_dr_backup_plan":						backupdr.DataSourceGoogleCloudBackupDRBackupPlan(),
-	"google_backup_dr_data_source":						backupdr.DataSourceGoogleCloudBackupDRDataSource(),
 	{{- end }}
+	"google_backup_dr_backup":							backupdr.DataSourceGoogleCloudBackupDRBackup(),
```

Comment:
these two data sources don't seem to contain any documentation. This comes from attempting to view it in the google-beta registry. Can you go ahead and include this as part of the PR?

Reply:
Just following up on this